lib/ui/painting.dart

```diff
@@ -2048,6 +2040,14 @@ class Codec extends NativeFieldWrapperClass1 {
 ///
 /// The returned future can complete with an error if the image decoding has
 /// failed.
+
+// Update this list when changing the list of supported codecs.
+/// {@template dart.ui.imageFormats}
+/// JPEG, PNG, GIF, Animated GIF, WebP, Animated WebP, BMP, and WBMP. Additional
```

Comment:
Will this not make this paragraph appear twice in the doc of `instantiateImageCodec`? Once here and another time above in line 2024 when the macro there gets expanded?

Reply:
Well, don't you see how they're documented?

https://github.com/dart-lang/dartdoc/wiki/Doc-comment-directives#templates-and-macros

😛 I think you are correct though. Updated.

Reply:
🤣